src/AwardType.php

```diff
@@ -16,7 +16,7 @@ abstract class AwardType
 
     public const PATREON_SUPPORTER = 6;
 
-    public static $active = [
+    public static array $active = [
```

Comment:
What's the difference between "static array" here and "const" for AchievementType::FLAGS? Shouldn't they both use the same syntax?

Reply:
Constants in PHP 8.0 cannot have a type - static properties should.
I was adding the missing type as the IDE complained about it.
Now that you mention it - it should be a const, too, as it's not to be changed during runtime.
I'll change that 👍

Reply:
See https://github.com/RetroAchievements/RAWeb/pull/986/commits/902d3b5b2c5216942025d1b4f5f30cb85017bdc2